### PR TITLE
Updating the 'Working with the Enhancement Tracking Board' part of the enhancements role handbook

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

More updates to the enhancements role handbook.
This PR adds instructions for interacting with the github project boards used to track enhancements.

#### Which issue(s) this PR fixes:

Part of #2109

### CODE REVIEW

File: release-team/role-handbooks/enhancements/README.md [hunk 1]
```diff
@@ -202,68 +202,65 @@ It is important that this process be followed and documentation remain up-to-dat
...
+Status tracks how if the enhancement has met all the criteria for inclusion in the current release.
+Refer to the [Enhancement Freeze Communications](#enhancement-freeze-templates) for detailed requirements.
 
-Once done, use the `Enhancements` -> `Remove Enhancements from Milestone` menu
-item to automatically to move it to the `Removed from Milestone` tab removing it
-from the `Dashboard`, `Enhancements` and `Docs` tabs.
+|                 Status | Description                                                                                                                                 |
+|-----------------------:|---------------------------------------------------------------------------------------------------------------------------------------------|
+|                Tracked | Enhancement has met all requirements for inclusion into current release.                                                                    |
+|              Untracked | ??? (remove?)                                                                                                                               |
```
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:253]: AFAIK we used `Pending Inclusion` this release.

File: release-team/role-handbooks/enhancements/README.md [hunk 2]
```diff
@@ -202,68 +202,65 @@ It is important that this process be followed and documentation remain up-to-dat
 - Start planning for the next release while assisting the Release Lead with anything relating to analytics or Public Relation planning of the release. Work with the Communications Lead to develop major themes for the official Kubernetes blog post.
 
 
-### Working with the Enhancement Tracking Sheet
+### Working with the Enhancement Tracking Board
 
-#### Cross Release Enhancement Tracking
+The Enhancements Tracking Board is used by various release sub-teams.
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:207]:
```suggestion
The Enhancements Tracking Board is used by the release-enhancements, release-docs and release-comms teams.
```

File: release-team/role-handbooks/enhancements/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:253]: Do you remember what `Pending Inclusion` is supposed to be used for?

File: release-team/role-handbooks/enhancements/README.md [hunk 3]
```diff
@@ -202,68 +202,65 @@ It is important that this process be followed and documentation remain up-to-dat
...
+- If the feature is graduating to `Alpha`, the type can either be Net New/Major Change. But usually when features are introduced to Kubernetes, they are not Major Changes.
+- If the feature is graduating to `Beta/Stable`, almost always the type is Graduating/Major Change. One exception to that is some features directly jump the hoop to Beta, in that case, the type can be Net New for even a `Beta` feature.
 
-If the Enhancement is being bumped to a later release, set it's state to
-`Deferred`.
+##### Status
 
-If it is being removed due to missing criteria or lack of response
-after being included in the milestone, set its state to `Removed from Milestone`.
+Status tracks how if the enhancement has met all the criteria for inclusion in the current release.
```
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:247]:
```suggestion
The field `Status` tracks if the KEP has met all the criteria for inclusion in the current release.
```

File: release-team/role-handbooks/enhancements/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:253]: `Removed from Milestone` === `Untracked` ?

**Comment by leonardpahlke** [comment]
it's maybe a little confusing to name a column `Status` and another one `Enhancements Status`?

File: release-team/role-handbooks/enhancements/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:253]: > Do you remember what Pending Inclusion is supposed to be used for?

@gracenng probably you know this one..

from my observation & understanding: If you are checking a KEP whether its ready or not the label can be applied.. its interim status, eventually the status gets updated to tracked or removed from milestone

**Comment by marosset** [comment]
> it's maybe a little confusing to name a column `Status` and another one `Enhancements Status`?

Agree. In the 1.27 board I made `Enhancements Status` into just `Status` and renamed the previous `Status` (which tracked Net New, Gradating, Major Change, or Deprecation) into `Type` to avoid have town columns with Status in the name.
I'm open to a better name for `Type`!

File: release-team/role-handbooks/enhancements/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:253]: I don't think we used `Untracked` in the past few releases either (we either set Status to `Deferred` or `Removed from Milestone`

I'm proposing that we remove `Pending Inclusion` and `Untracked` from Status options unless we can clearly define when they should be used.

File: release-team/role-handbooks/enhancements/README.md (hunk 2 above)
Comment (marosset) [reply release-team/role-handbooks/enhancements/README.md:207]: Can you add an oxford comma in your suggestion?

Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:207]: 👍

**Comment by marosset** [comment]
@leonardpahlke @salaxander can one of you take another look at this?
I'd be good to get these updates merged before the new shadows start onboarding.

File: release-team/role-handbooks/enhancements/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:338]: > I'm proposing that we remove Pending Inclusion and Untracked from Status options unless we can clearly define when they should be used.

Sounds ok to me. We can add it back if we should need it.. 👍

[the remark this answers is not in the document]
Comment (leonardpahlke) [reply release-team/role-handbooks/enhancements/README.md:338]:
```suggestion
```

File: release-team/role-handbooks/enhancements/README.md [hunk 4]
```diff
@@ -287,68 +287,65 @@ If your SIG is not planning to submit an enhancement, please reply to this messa
...
+|              Untracked | ??? (remove?)                                                                                                                               |
+|      Pending Inclusion | ??? (remove?)                                                                                                                               |
+|                At Risk | Enhancement does not currently meet requirements for inclusion into current release.                                                        |
+|     Exception Required | Enhancement did not meet requirements by enhancement / code freeze and deadlines for requesting an [exception](#exceptions) has not passed. |
+|               Deferred | Enhancement has been bumped to a future release by owning SIG.                                                                              |
+| Removed From Milestone | Enhancement did not meet requirements by enhancement / code freeze for current release and an exception was not requested and/or approved.  |
 
-#### Moving a Removed Enhancement Back into the Milestone
+Notes:
```
Comment (leonardpahlke) [inline release-team/role-handbooks/enhancements/README.md:345]:
```suggestion
Tipps:
```
or smth similar maybe to be more precise (none block)

**Comment by marosset** [comment]
@leonardpahlke @salaxander - is there any other feedback here? I'd like to get this merged being starting to onboard enhancement shadows. I can always fix things in future PRs too

**Pull request merged**